axiomapy/axiomaapi/analyses.py

```diff
@@ -870,3 +870,87 @@ def post_precompute(
             return_response=return_response,
         )
         return response
+
+    @staticmethod
+    def get_brinson_asset_returns(
+        request_id: int,
+        headers: dict = None,
+        return_response: bool = False,
+    ):
+        """This method returns Brinson Asset Returns report from the attribution analysis with the given request id
+
+        Args:
+            request_id: Request id for the analysis request
+            headers: Optional headers, if any required (Correlation ID , Accept-Encoding)
+            return_response: If set to true, the response will be returned
+
+        Returns:
+            Brinson Asset Returns in json or csv format
+        """
+        url = f"/analyses/performance/{request_id}/results/brinson-asset-returns"
+        _logger.info(f"Getting from {url}")
+        response = AxiomaSession.current._get(
+            url, headers=headers, return_response=return_response,
+        )
+        return response
+
+    @staticmethod
+    def get_summary_time_series(
+            request_id: int,
+            headers: dict = None,
+            return_response: bool = False,
+            report_frequency: str = "Default"
```

Comment:
Can you share some details on what the frequency enum will accept?

Reply:
Updated the description. Please let me know if this looks better!